canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -206,9 +208,10 @@ $palette-border-color: $ui-03;
 .palette-flyout-category-item {
 	display: flex;
 	align-items: center;
+	user-select: none;
 	/* Prevent elements from being selectable */
 	/* This stops it being dragged incorrectly.*/
-	user-select: none;
```

Comment:
Why was this removed. It prevents text from being selected (highlighted) by accident. Unless there is a good reason can you put it back please. Also, the comment above goes with the line of code so it the line of code is removed the comment also has to be removed.